.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Stale issue message'
+        stale-pr-message: 'Stale pull request message'
+        stale-issue-label: 'no-issue-activity'
```

Comment:
We have a label for stale `kind/stale` can that be used here?

Can we change the messages to relay the reason that it is being marked stale?

Reply:
i've updated the label `kind/stale` to apply to both stale issues and PRs. 

I can add messaging around what it means to be stale ie. "This issue/PR is being marked as stale due to a long period of inactivity"
thoughts @jharshman ?

Reply:
@xchapter7x Yes I think that would be helpful.

Reply:
message is not updated to be more relevant.